corehq/tabs/tabclasses.py

```diff
@@ -946,8 +947,8 @@ def dropdown_items(self):
             ))
         if toggles.MANAGE_CCZ_HOSTING.enabled_for_request(self._request):
             submenu_context.append(dropdown_dict(
-                ManageHostedCCZ.page_title,
-                url=reverse(ManageHostedCCZ.urlname, args=[self.domain])
+                ManageHostedCCZ_page_title,
+                url=reverse(ManageHostedCCZ_urlname, args=[self.domain])
```

Comment:
Did you consider other models for this?  It'll be a bit weird for this feature flag and link to exist, but point to a broken link.  I'm guessing there will be a number of things like this, so I could see something like `try: import x` or even a plugin architecture, something like how `CUSTOM_REPORTS` is used (though it'd probably mean moving this link to somewhere else).  I think fine to punt for now and reconsider if it becomes a more widespread thing.

Reply:
I did spend a bit of time considering other options but I didn't wan't to invest a lot of time in a plugin framework until I knew the scope of the problem.

What do you think about using signals in a similar way to what we did for domain deletions: https://github.com/dimagi/commcare-hq/commit/78db50c38196fd50333170bd8bd975bf37b37439

Reply:
Interesting idea.  I think of signals as performing an action and possibly reporting on the outcome rather than as returning a result, but they _are_ easily extensible.  It could also be a map stored in `settings`.  No strong feelings.